src/collaborative_platform/apps/help/static/help/tei_sample_document.xml

```diff
@@ -0,0 +1,156 @@
+              <surname>Maguire</surname>
+            </persName>
+            <name>Jerry Maguire</name>
+            <birth when="2001-02-03"/>
+          </person>
+          <person resp="#recipe_levenshtein_matching" xml:id="person-1">
+            <name>Ingwer</name>
+          </person>
+          <person resp="#recipe_levenshtein_matching" xml:id="person-2">
+            <name>Grieben</name>
+          </person>
+          <person xml:id="person-3">
+            <name>Met</name>
+          </person>
+        </listPerson>
+      </div>
+      <div type="event">
+        <listEvent type="eventList">
+          <event xml:id="event-0">Big Bang</event>
+        </listEvent>
+      </div>
+      <div type="place">
+        <listPlace type="placeList">
+          <place xml:id="place-0">
+            <country>Poland</country>
+            <settlement>Poznań</settlement>
+            <location>
+              <geo>52.408333 16.934167</geo>
+            </location>
+            <placeName>
+              <name>PCSS</name>
+            </placeName>
+          </place>
+        </listPlace>
+      </div>
+      <div type="ingredient">
+        <listObject type="ingredientList">
+          <object type="ingredient" xml:id="ingredient-3">Met</object>
+        </listObject>
+      </div>
+      <div type="utensil">
+        <listObject>
+          <object type="utensil" xml:id="form">Form</object>
+        </listObject>
+      </div>
+      <div type="dietetic">
+        <listObject/>
+      </div>
+      <div type="recipe">
+        <p>Put three or four Pounds of <name xml:id="name-4" ref="#person-0">Jerry Maguire's</name> coarse Beef , with <name xml:id="name-17" ref="#event-0">three Pints of Pease</name> , into two <ab xml:id="ab-0">Gallons</ab> of <name xml:id="name-3" ref="#person-0">Hank Moody's</name> Water ; let it boil till the <name xml:id="name-7" ref="#ingredient-3">Meat</name> is all to <name xml:id="name-8" ref="#ingredient-0">Rags</name> ; and Half an Hour before you strain it out , put in two or three <name xml:id="name-0" ref="#place-0">Anchovies</name> ; then strain it from the Meat and Husks , and put in as much as you want <name xml:id="name-15" ref="#form">for</name> the Meal into a Sauce-pan , with an Onion stuck with Cloves , a Race of <name xml:id="name-2" ref="#ingredient-1">Ginger</name> bruis 'd , a little Fagot of Thyme , Savory , Parsley , and a little Pepper . Let it boil near Half an Hour ; stir in a Piece of Butter , and fry some forc 'd Meat Balls , Bacon and French Bread cut in Dice , with Spinage boil 'd green , to put to it in the Dish .</p>
+        <p>Some random date <date xml:id="date-0" name="sam's birthday" ref="#date-0" when="2020-07-04">4th of july 2010</date> for test unlistable entities, and reference to this date <date xml:id="date-1" ref="#date-0">the exact same day</date>. Another important date <date xml:id="date-66" ref="#date-66" when="1791-05-03">3rd of may 1791</date>.</p>
```

Comment:
_name_ parameter is not permitted within _date_ tag :(

Reply:
It is now not permitted indeed. This file, however, is taken from the ones provided for testing as a placeholder. 
It is my intention to create a file specifically for this where the content is a brief introduction to the project, platform
and that uses all entities and annotations.

Reply:
I will update this one and, later on, work on this better alternative I have described.